GarupaPico/GarupaPico.Android/GarupaPico.Android.csproj

```diff
@@ -103,7 +103,7 @@
     </AndroidResource>
   </ItemGroup>
   <ItemGroup>
-    <AndroidAsset Include="Assets\calculator-graphic.png" />
+    <Content Include="calculator-graphic.png" />
```

Comment:
Read the docs: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/images?tabs=windows
Android images should be placed under `Resources/drawable`